src/app/clusters/bindings/bindings.cpp

```diff
@@ -191,6 +193,7 @@ CHIP_ERROR BindingTableAccess::WriteBindingTable(const ConcreteDataAttributePath
         {
             CreateBindingEntry(iter.GetValue(), path.mEndpointId);
         }
+        LogErrorOnFailure(NotifyBindingsChanged());
```

Comment:
Shouldn't this happen from inside the binding table?  Things other than the cluster can modify the binding table....

Reply:
If we did, then it would generate far more events. Would you prefer that I change the name of the event from BindingChanged to ClusterBindingChanged?

Reply:
I think we should be very clear about what this event means and why we are emitting it.  If we want to find out when there are new bindings, then why do we care whether they were added by the cluster bits or by the app itself?